code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -38,8 +39,8 @@
 	icon_state = "mouse_[body_color]"
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
-	if(prob(40))
-		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(2, 4))
+	if(prob(75))
+		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(3, 6), 9, rand(3,4))
```

Comment:
same here

Reply:
ok this one tho. if i revert this one it will scream gitbot will scream merge conflict.